tareas/clase-6/tarea-2-clase-6.js

```diff
@@ -0,0 +1,126 @@
+
+document.querySelector("#boton-agregar").onclick = function() {
+    
+    const nodoPagina = document.querySelector("form");
+    const nuevoLabel = document.createElement("label");
+    const nuevoInput = document.createElement("input");
+    const textoLabel = document.createTextNode("Ingrese sueldo mensual");
+    const botonCalcular = document.querySelector("#boton-calcular");
+  
+
+    nuevoLabel.appendChild(textoLabel);
+    nuevoLabel.appendChild(nuevoInput);
+    nodoPagina.appendChild(nuevoLabel);
+    botonCalcular.style.display = 'inline-block';
+    nuevoInput.setAttribute("class", "sueldos")
+	nuevoInput.type = "number";
+		
+    
+    return false
+
+};
+
+document.querySelector("#boton-calcular").onclick = function() {
+
+    const resultados = document.querySelector("#resultados");
+    const sueldosMensuales = document.querySelectorAll(".sueldos");
+    
+    resultados.style.display = "inline-block";
+
+    let sueldosAnuales = calcularSueldosAnuales(sueldosMensuales);
+    
+    
+
+    function calcularPromedioMensual(sueldosMensuales) {
+		
+        let contador = 0;
+
+		for(let i =0; i < sueldosMensuales.length; i++) {
+			contador += Number(sueldosMensuales[i].value);
+		}
+
+		const salarioPromedioMensual = contador / sueldosMensuales.length;
+		
+		document.querySelector("#salario-promedio-mensual").textContent = salarioPromedioMensual;
+	};
```

Comment:
todo esto está mal indentado, fijate que parece que el `for` está dentro del `let`